Doppler.PushContact/Services/IPushContactService.cs

```diff
@@ -27,6 +27,8 @@ public interface IPushContactService
 
         Task<ApiPage<DomainInfo>> GetDomains(int page, int per_page);
 
+        Task<MessageDeliveryResult> GetHistoryEventResultByMessageIdAsync(string domain, Guid messageId);
```

Comment:
What do you think change the name to indicate that it return the summarization of the delivered message? Something like:
```suggestion
        Task<MessageDeliveryResult> GetDeliveredMessageSummarizationAsync(string domain, Guid messageId);
```

Reply:
Sounds good, i will change it.